db.js

```diff
@@ -435,7 +439,16 @@ export class ActivityPubDB extends EventTarget {
     console.log(note.replies)
     if (note.replies && typeof note.replies === 'string') {
       console.log('Attempting to load replies for:', note.id)
-      await this.ingestActivityCollection(note.replies, note.attributedTo, true)
+      try {
+        await this.ingestActivityCollection(note.replies, note.attributedTo, true)
```

Comment:
The collection is not activities, this would only show replies by the author of the note instead of *all* replies.

Instead it should be something like this:

```javascript
async ingestReplies(url) {
  for await (const note of this.iterateCollection(replies, {limit: Infinity})) {
    await this.ingestNote(note)
  }
}
```

then you can also get rid of the `note.replies.items` if statement below as well.